dev/lnd/lnd.conf

```diff
@@ -26,3 +26,4 @@ noseedbackup=1
 bitcoind.rpchost=bitcoind:18443
 keysend-hold-time=2s
 tlsextradomain=lnd
+routing.assumechanvalid=true
```

Comment:
maybe you can add a comment on why this is necessary?

Reply:
uhmm is not necessary, that is why I had it in the readme (optional). This just improve performance in regtest (for slow environments).

https://github.com/GaloyMoney/galoy/commit/6405301fcf880eb2c62a326d60811a5db8a3f67e

Reply:
ref: https://github.com/lightningnetwork/lnd/blob/6a97e6432d8d003752d4827a4a8bf05d16b279b6/rpcserver.go#L2523 let me know what to do... cc @bodymindarts

Reply:
I like adding a comment in the dev lnd.conf just to be clear that this is not a setting that should ever be used in prod incase someone copies the config. So a hint why it makes sense in dev but shouldn't be used anywhere else would be good.